### PR TITLE
taxonomy: Teewurst

### PR DESCRIPTION
**Pull request by moon-rabbitOFF**
### Related issue(s) and discussion

- Fixes #[ISSUE NUMBER]

### CODE REVIEW

File: taxonomies/food/categories.txt [hunk 1]
```diff
@@ -87064,6 +87071,18 @@ en: Liverwurst sausages
 < en:Meat spreads
 en: Liverwurst spreads
 
+< en:Prepared meats
+xx: Teewurst
```
Comment (stephanegigandet) [inline taxonomies/food/categories.txt:87075]: For categories, we put en: as the canonical name first, so it should just be:

en: Teewurst
de: Teewurst
xx: Teewurst

File: taxonomies/food/categories.txt [hunk 2]
```diff
@@ -49460,16 +49460,22 @@ ciqual_food_code:en: 19666
 ciqual_food_name:en: Petit-Suisse like fresh cream cheese, plain, around 9% fat
 ciqual_food_name:fr: Fromage frais type petit suisse, nature, 10% MG environ
 
+# Half milk and half cream (or Half and half) is a common preparation in the USA which is used in coffee or with breakfast cereals
+< en:Dairies
+en: Half milk and half cream
```
Comment (stephanegigandet) [inline taxonomies/food/categories.txt:49465]: Could you add "Half and half" as a synonym?

Comment (moon-rabbitOFF) [reply taxonomies/food/categories.txt:49465]: I added it!

File: taxonomies/food/categories.txt (hunk 1 above)
Comment (moon-rabbitOFF) [reply taxonomies/food/categories.txt:87075]: I changed it!

File: taxonomies/food/categories.txt (hunk 2 above)
Comment (stephanegigandet) [reply taxonomies/food/categories.txt:49465]: Thank you!

**Review by stephanegigandet — APPROVED** [review]
Thanks

**Pull request merged**